src/man/sssd-ldap.5.xml

```diff
@@ -551,6 +551,26 @@
                     </listitem>
                 </varlistentry>
 
+                <varlistentry>
+                    <term>ldap_connection_idle_timeout (integer)</term>
+                    <listitem>
+                        <para>
+                            Specifies a timeout (in seconds) that an idle
+                            connection to an LDAP server will be maintained.
+                            If the connection is idle for more than this time
+                            then the connection will be closed.
```

Comment:
Let's add a note that 0 disables a feature?

Reply:
Ok, done.